sonar/modules/institutions/jsonschemas/institutions/institution-v1.0.0.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://sonar.ch/schemas/institutions/institution-v1.0.0.json",
+  "additionalProperties": false,
+  "title": "SONAR institution v1.0.0",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "default": "https://sonar.ch/schemas/institutions/institution-v1.0.0.json"
+    },
+    "pid": {
+      "description": "Institution identifier.",
```

Comment:
please use title instead of description same above

Reply:
I corrected the titles and descriptions